### DISCUSSION TITLE
What is the difference beteen parquet_scan and read_parquet

### DISCUSSION THREAD

**Discussion by geoHeil** [duckdb/duckdb#3421] [Q&A] [answered]
What is the difference between parquet_scan and read_parquet functions?

`SELECT * FROM read_parquet(['file1.parquet', 'file2.parquet', 'file3.parquet']);`

the new documentation only seems to recommend https://duckdb.org/docs/data/parquet the latter.

**Comment by hannes** [comment, marked as answer]
There is no difference, they are just aliases of each other. Code here: https://github.com/duckdb/duckdb/blob/f44e77c7ec356ca6aee96dc84b0e00a6d3c6973a/extension/parquet/parquet-extension.cpp#L509

**Reply by geoHeil** [reply]
Thanks! So is my understanding correct that the `create or replace view foo as select * from read_parquet(' /path/to/foo.parquet')` will only store the reference to the potentially large file in duckDB and not transform the raw parquet in the duck-DB binary database file?

**Reply by Mytherin** [reply]
Yes that is correct.
